src/components/SearchPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Search } from 'react-feather';
+import { Colors, blueGradientBgImg } from '../../constants';
+import { SearchBar } from '../SearchBar';
+import { TitleBar } from '../TitleBar';
+import { FilterItemList, FilterItemListProps } from '../FilterItemList';
+
+export interface SearchPageProps {
+  pageTitle: string;
+  results: FilterItemListProps;
+}
+
+const style = {
+  ...blueGradientBgImg,
+  maxWidth: '1024px'
+};
+
+export const SearchPage = (props: SearchPageProps): JSX.Element => (
+  <div>
+    <div style={style}>
+      <TitleBar claimTitle={props.pageTitle} />
+    </div>
+    <SearchBar />
```

Comment:
Please add style to center the search bar over the results and give the search bar a bit of space between it the filter and the results.